src/matrix.jl

```diff
@@ -525,3 +525,28 @@ function ∩(A::IntervalMatrix, B::IntervalMatrix)
     end
     return C
 end
+
+"""
+    diam_norm(A::IntervalMatrix{T}, p=Inf) where {T}
+
+Return the diameter norm of the interval matrix.
+
+### Input
+
+- `A` -- interval matrix
+- `p` -- (optional, default: `Inf`) the `p`-norm used; valid options are:
+         `1`, `2`, `Inf`
```

Comment:
I think for `p=2` this crashes. Can you confirm that it works?

Reply:
It works:

```julia
julia> m = rand(IntervalMatrix)
2×2 IntervalMatrix{Float64,Interval{Float64},Array{Interval{Float64},2}}:
 [-0.205295, 0.430324]  [-0.807889, 0.621021]
  [0.124467, 1.40682]    [0.156383, 0.45967]

julia> diam_norm(m, 2)

1.8409122040535542
```
From the docs of `opnorm`,

>   Compute the operator norm (or matrix norm) induced by the vector p-norm, where valid values of p are 1, 2, or Inf. (Note that for sparse matrices, p=2 is
  currently not implemented.) Use norm to compute the Frobenius norm.

Reply:
Oh, sorry, it is applied to a normal matrix. Sure, then it works :+1: